package-res/resources/web/pentaho/visual/base/model.js

```diff
@@ -45,37 +73,287 @@ define([
      * @description Creates a base `Model`.
      * @param {pentaho.visual.base.spec.IModel} modelSpec A plain object containing the model specification.
      */
-    var Model = Complex.extend({
-          meta: {
-            id: "pentaho/visual/base",
-            view: "View",
-            isAbstract: true,
-            props: [
-              {
-                name: "width",
-                type: "number",
-                isRequired: true
-              },
-              {
-                name: "height",
-                type: "number",
-                isRequired: true
-              },
-              {
-                name: "isInteractive",
-                type: "boolean",
-                value: true
-              },
-              {
-                name: "data",
-                type: "object",
-                isRequired: true
-              }
-            ]
+    var Model = Complex.extend().implement(EventSource).extend(/** @lends pentaho.visual.base.Model# */{
```

Comment:
I see that, but why would you do it temporarily here instead of:
1. temporarily in EventSource
2. simply putting a note in the calling code saying it should switch to a future "_emitSafe" when one exists?

Don't think this is a good choice. It definitely, at a minimum, requires a note saying it is a temporary substitute for the thing that does not yet exist in EventSource.